docs/_data/table/en/common-configuration.yml

```diff
@@ -98,6 +98,9 @@ alluxio.underfs.s3a.consistency.timeout.ms:
 alluxio.underfs.s3a.directory.suffix:
   Directories are represented in S3 as zero-byte objects named with the specified suffix. The
   default value is "/".
+alluxio.underfs.s3a.list.objects.version:
+  The version of GET Bucket (List Objects) API to use. Supported values are "v1" and "v2". The
+  default value is "v2".
```

Comment:
I would prefer to make this a boolean. If there turn out to be multiple API versions, we should have different connectors instead of having one connector with many switches.